### ISSUE TITLE
[BUG] kirby input with type=date inside modal on mobile device

### ISSUE THREAD

**Issue by tspringborg** [kirbydesign/designsystem#2367]
## Describe the bug
See attached video. Notice the jumping and how a translate is added to the input.

https://user-images.githubusercontent.com/1595605/175253469-4101f304-604b-462d-99c8-a71fc0546cc4.mov

## Describe how to reproduce the bug
Show a kirby modal with a component that contains 
```
<kirby-form-field label="Date input" >
  <input kirby-input type="date" />
</kirby-form-field>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
<p>spacing so it's scrollable</p>
```

Emulate phone in browser and use the simulated touch curser to scroll while input field is selected and has some content.

## Which Kirby version was used?
5.4.1

## What was the expected behavior?
Input field not jumping and preferably not getting transform styles attached.

## Please complete the following information:
- OS: MacOS Monterey 12.3.1
- Browser: Chrome
- Browser version: 102.0.5005.115

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
